src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/request/VideoLinkBookingCreateRequest.kt

```diff
@@ -0,0 +1,94 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.Valid
+import jakarta.validation.constraints.AssertTrue
+import jakarta.validation.constraints.Future
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.NotEmpty
+import jakarta.validation.constraints.NotNull
+import jakarta.validation.constraints.Size
+import java.net.URI
+import java.time.LocalDate
+import java.time.LocalTime
+
+data class VideoLinkBookingCreateRequest(
+
+  @field:NotNull(message = "The video link booking type is mandatory")
+  @Schema(description = "The booking type", example = "COURT")
+  val bookingType: BookingType?,
+
+  @field:Valid
+  @field:NotEmpty(message = "At least one prisoner must be supplied for a video link booking")
+  @Schema(description = "The prisoner or prisoners associated with the video link booking")
+  val prisoners: List<PrisonerDetails>,
+
```

Comment:
We might limit to one initially but good that its a list - for later (co-defendants in mind)>.

Reply:
For simplicity shall I change it to one?

Reply:
No I don't think so. The booking journey will limit to one anyway - and in the background we can keep in mind that it might expand to more than one.